### PR TITLE
File sizes

### PR DESCRIPTION
**Pull request by jackhannah95**
This is mostly creating `file_size`, some tests for it, and adding some files used in the tests. Thinking of tests for this was quite hard so let me know if you have any good ideas.

I also created a `phimethods.R` script for some package-level documentatin. It's mainly just to include the line `if(getRversion() >= "2.15.1")  utils::globalVariables(c("."))` which stops a note appearing during `R CMD check` about `.` being an undefined variable.

Also added some code from @alan-y to `.travis.yml` to test on old and development versions of R.

### CODE REVIEW

**Comment by alan-y** [comment]
one option to test is to create a directory within the `inst` folder of your package, e.g. `inst/test_files` and then create put a few test files in there. Hopefully you should be able to then use these files to create unit tests - otherwise you will need to experiment with the mockery package which should be able to _trick_ the unit tests into believing there are files actually there when there are not (see https://blog.r-hub.io/2019/10/29/mocking/)

**Comment by jackhannah95** [comment]
I basically just copied the way [readr](https://github.com/tidyverse/readr/tree/master/tests/testthat) and [readxl](https://github.com/tidyverse/readxl/tree/master/tests/testthat) do their tests by saving all the files inside the `testthat` folder and using `testthat::test_path` to refer to them. It seems to work alright so hopefully nothing complicated like mocking will be required, but I found this quite a hard function to think up tests for.

**Comment by alan-y** [comment]
ah didn't know about that, sounds like the best solution so will keep it in mind cheers!

File: R/file_size.R [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+#' `Word`
+#' \item \code{.rds} files are prefixed with `RDS`
+#' \item \code{.txt} files are prefixed with `Text`,
+#' \item \code{.fst} files are prefixed with `FST`,
+#' \item \code{.pdf} files are prefixed with `PDF`,
+#' \item \code{.tsv} files are prefixed with `TSV`,
+#' \item \code{.html} files are prefixed with `HTML`,
+#' \item \code{.ppt}, \code{.pptm} and \code{.pptx} files are prefixed with
+#' `PowerPoint`,
+#' \item \code{.md} files are prefixed with `Markdown`
```
Comment (davidc92) [inline R/file_size.R]: Rmarkdown files also get the "Markdown" prefix so maybe include that here?

**Pull request merged**